OptimizelySDK.Tests/EventTests/CanonicalEvent.cs

```diff
@@ -0,0 +1,83 @@
+﻿using OptimizelySDK.Entity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OptimizelySDK.Tests.EventTests
+{
+    public class CanonicalEvent
+    {
+        private string ExperimentId;
+        private string VariationId;
+        private string EventName;
+        private string VisitorId;
+        private Dictionary<string, object> Attributes;
+        private EventTags Tags;
+
+        public CanonicalEvent(string experimentId, string variationId, string eventName, string visitorId, Dictionary<string, object> attributes, EventTags tags)
+        {
+            ExperimentId = experimentId;
+            VariationId = variationId;
+            EventName = eventName;
+            VisitorId = visitorId;
+            Attributes = attributes ?? new Dictionary<string, object>();
+            Tags = tags ?? new EventTags();
```

Comment:
I feel the file for EventTags should be called `EventTags` and not `EventAttributes`.